source/applications/dependency-injection.md

```diff
@@ -191,11 +191,17 @@ export default Ember.Component.extend({
 
 ## Factory Instance Lookups
 
-The vast majority of Ember registrations and lookups are performed implicitly.
+You rarely need to write code to look up objects in an Ember application because
```

Comment:
"look up objects" seems both vague and like insider baseball here to me. Look up code from where? When might I be looking objects? I can imagine a developer new to Ember reading this and being confused.